framework-crates/objc2-foundation/src/thread.rs

```diff
@@ -41,7 +41,33 @@ pub fn is_multi_threaded() -> bool {
 /// Whether the current thread is the main thread.
 #[cfg(feature = "NSThread")]
 pub fn is_main_thread() -> bool {
```

Comment:
Nit: Could you `#[inline]` this and the `imp` functions, so that they can be inlined even without LTO?

Reply:
And also, I'm doing merge commits in this repo, so if you could squash, that'd be nice!